ClearentIdtechIOSFramework/ClearentUI/ReaderDetails/ClearentReaderDetailsPresenter.swift

```diff
@@ -59,6 +60,13 @@ class ClearentReaderDetailsPresenter: ClearentReaderDetailsProtocol {
         ClearentWrapper.shared.removeReaderFromRecentlyUsed(reader: currentReader)
         handleBackAction()
     }
+    
+    func updateReader(reader: ReaderInfo) {
+        if (reader.readerName == ClearentWrapperDefaults.pairedReaderInfo?.readerName) {
```

Comment:
since `==` operator is overloaded, we can directly compare 2 instances of ReaderInfo like this:
```suggestion
        if (reader == ClearentWrapperDefaults.pairedReaderInfo) {
```
check https://github.com/clearent/ClearentIdtechIOSFramework/blob/enhanced_sdk/ClearentIdtechIOSFramework/ClearentUI/PaymentFlowDataSource/ReaderInfo.swift#L34